app/views/content_items/_detailed_guide_b.html.erb

```diff
@@ -0,0 +1,54 @@
+<%= content_for :title, @content_item.page_title %>
```

Comment:
To someone editing a detailed guide or case study, it is not clear that they need to make the change in two places.

Why does it need to be clear that there is a big switch?